properties.go

```diff
@@ -0,0 +1,62 @@
+package notmuch
+
+// Copyright © 2015 The go.notmuch Authors. Authors can be found in the AUTHORS file.
+// Licensed under the GPLv3 or later.
+// See COPYING at the root of the repository for details.
+
+// #cgo LDFLAGS: -lnotmuch
+// #include <stdlib.h>
+// #include <notmuch.h>
+import "C"
+
+// Properties represent a notmuch properties type.
+type Properties cStruct
```

Comment:
Perhaps we should call this `MessageProperties`? I could conceive of the C library adding e.g. `notmuch_thread_properties_t` later on.